f8a_report/s3_helper.py

```diff
@@ -12,14 +12,35 @@
 class S3Helper:
     """Helper class for storing reports to S3."""
 
-    def __init__(self):
+    def __init__(self, aws_access_key_id=None, aws_secret_access_key=None, report_bucket=None):
         """Init method for the helper class."""
-        self.region_name = os.environ.get('AWS_S3_REGION') or 'us-east-1'
-        self.aws_s3_access_key = os.environ.get('AWS_S3_ACCESS_KEY_ID')
-        self.aws_s3_secret_access_key = os.environ.get('AWS_S3_SECRET_ACCESS_KEY')
+        self.region_name = 'us-east-1' or os.environ.get('AWS_S3_REGION')
```

Comment:
This should be 
`self.region_name =os.environ.get('AWS_S3_REGION') or  'us-east-1' `

Reply:
Same for all the below statements. We should pick env variable first followed by some default value

Reply:
Done.